latest_artifact.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Shell seems to not like the `<<<` indirect
```
Syntax error: redirection unexpected
```
I think `bash` is out of beta testing.